File: SampleSetup/Protocol/Body/ApiUserRegisterReturn.cs
using System.Collections.Generic;
using SampleProject.Protocol.Context;

namespace SampleProject.Protocol.Body
{
    [UsingRef(nameof(AbstractReturn))]
    [UsingRef(nameof(ApiTokenPostReturn))]
    [UsingRef(nameof(CurrencyTypes), "..", nameof(Context))]
    public class ApiUserRegisterReturn : AbstractReturn
    {
        public int UserCurrencyTypeEnumId { get; set; } = (int)CurrencyTypes.EUR;

        public ApiTokenPostReturn? Token { get; set; } = null;

        public List<string>? ErrorCodes { get; set; } = null;
    }
}

File: SampleSetup/Output/PHP/SampleProject/Protocol/Body/ApiUserRegisterReturn.php
<?php
namespace SampleProject\Protocol\Body;

class ApiUserRegisterReturn extends \SampleProject\Protocol\Body\AbstractReturn {
    /** @var int */ public $UserCurrencyTypeEnumId;

    /** @var \SampleProject\Protocol\Body\ApiTokenPostReturn */ public $Token;

    /** @var string[] */ public $ErrorCodes;

    public function __construct()
    {
        $this->UserCurrencyTypeEnumId = \SampleProject\Protocol\Context\CurrencyTypes::EUR;
    }
}
